src/main/java/core/basesyntax/NewYearDiscountService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;

```

Comment:
```suggestion
package core.basesyntax.strategy.impl;
```

same for other strategy implementations